pkg/api/admin/platformworkloadidentityroleset_convert.go

```diff
@@ -21,12 +20,17 @@ func (c platformWorkloadIdentityRoleSetConverter) ToExternal(s *api.PlatformWork
 		},
 	}
 
-	for i, r := range s.Properties.PlatformWorkloadIdentityRoles {
-		out.Properties.PlatformWorkloadIdentityRoles[i].OperatorName = r.OperatorName
-		out.Properties.PlatformWorkloadIdentityRoles[i].RoleDefinitionName = r.RoleDefinitionName
-		out.Properties.PlatformWorkloadIdentityRoles[i].RoleDefinitionID = r.RoleDefinitionID
-		out.Properties.PlatformWorkloadIdentityRoles[i].ServiceAccounts = make([]string, 0, len(r.ServiceAccounts))
-		out.Properties.PlatformWorkloadIdentityRoles[i].ServiceAccounts = append(out.Properties.PlatformWorkloadIdentityRoles[i].ServiceAccounts, r.ServiceAccounts...)
+	for _, r := range s.Properties.PlatformWorkloadIdentityRoles {
+		role := PlatformWorkloadIdentityRole{
```

Comment:
Nit: space between `PlatformWorkloadIdentityRole` and `{`

Reply:
I'd put a space before the curly brace if for example this were line 23 and I had left out the space after the `range` expression, but I think it's normal Go style to avoid leaving blank space in that spot in a struct literal, and that's what's done throughout the codebase.